app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -1,5 +1,9 @@
-<p>Welcome <%= @email %>!</p>
+<% if !@user.has_role?(:creator) %>
```

Comment:
You may plan on doing this at a later stage so apologies if I'm being too proactive 😅 

We can move this into a method on the User model ([similar to this this](https://github.com/forem/forem/blob/main/app/models/user.rb#L323-L325))

Reply:
Waiting on it to be merged, but that change is in #14883.